flink-runtime/src/test/java/org/apache/flink/runtime/leaderretrieval/SettableLeaderRetrievalServiceTest.java

```diff
@@ -32,6 +32,7 @@
 public class SettableLeaderRetrievalServiceTest extends TestLogger {
 
     private SettableLeaderRetrievalService settableLeaderRetrievalService;
+    private static final long TIMEOUT = 50L;
```

Comment:
```suggestion
    private static final long TIMEOUT_IN_MILLIS = 50L;
```
It's always a good habit to add the unit to variables like that. Alternatively, you could use Duration here and do initialize it with `Duration.ofHours()` to inialize it. In the test, you can then do `TIMEOUT.toMillis()`. That makes it clearer to the reader.

Reply:
On another note: Having low timeouts increases the risk of having flaky tests. I'm wondering whether we should go to something higher like 1 hour.

Reply:
Thanks for the suggestion - I believe a few other tests are also using a timeout like this, i'll bring it in line with more what they're doing.

Reply:
Actually, I prefer your suggestion of using a `Duration.ofHours` for clarity. Will push up a change.